engine/table/src/main/java/io/deephaven/engine/table/impl/lang/QueryLanguageFunctionUtils.java

```diff
@@ -19276,31 +19278,31 @@ public static byte byteCast(short a) {
     }
 
     public static int intCast(Object a) {
-        return a == null ? QueryConstants.NULL_INT : (int) a;
+        return a == null ? QueryConstants.NULL_INT : ((Number) a).intValue();
```

Comment:
please change the query lang func utils generator, not this class (then run it, and it will re-generate this class).

note that @jcferretti is also making changes to that file, perhaps coordinate with him to be sure you aren't changing overlapping code.

Reply:
As discussed, ignore, Cristian is handling this part too.